app/lib/search/search_service.dart

```diff
@@ -218,6 +221,7 @@ class SearchQuery {
   final String query;
   final ParsedQuery parsedQuery;
   final String platform;
+  final List<String> owners;
```

Comment:
Can we document this? also wouldn't it be better to call this `ownersIsOneOf` or something more descriptive?

Reply:
Do we expect to have queries where we have owners with the semantics of all-of-them? I think there may be a query where we want to know which packages we have a shared ownership of, but the use-case is probably not that common. If that is not something we want to have here, let's keep it short.

Reply:
Since this is an internal thing, maybe it's better to keep it long and descriptive... I do like short names, but this is also sort of a hack because we don't have OR support in queries... so we make this work as a hack.

We can make it clear that this is a workaround for not having or... by writing `ownerIsOneOf` or `ownedByOneOf`... as this won't conflict with a future filter we might add..